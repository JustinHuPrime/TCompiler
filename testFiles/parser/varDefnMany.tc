module foo;

int bar = 12, baz, qux = 0;