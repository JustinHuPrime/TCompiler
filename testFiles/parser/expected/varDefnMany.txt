testFiles/parser/varDefnMany.tc (code):
FILE(1, 1, STAB(ENTRY(qux, VARIABLE(testFiles/parser/varDefnMany.tc, 3, 20, int)), ENTRY(bar, VARIABLE(testFiles/parser/varDefnMany.tc, 3, 5, int)), ENTRY(baz, VARIABLE(testFiles/parser/varDefnMany.tc, 3, 15, int))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), VARDEFN(3, 1, KEYWORDTYPE(3, 1, int), ID(3, 5, bar, REFERENCES(testFiles/parser/varDefnMany.tc, 3, 5)), ID(3, 15, baz, REFERENCES(testFiles/parser/varDefnMany.tc, 3, 15)), ID(3, 20, qux, REFERENCES(testFiles/parser/varDefnMany.tc, 3, 20)), LITERAL(3, 11, UBYTE(12)), (null), LITERAL(3, 26, UBYTE(0))))
